KMB/make_KMB_info.py

```diff
@@ -171,6 +174,27 @@ def get_photographer_mapping(self, photographer_page):
                 qid, photographer_props, self.photographer_cache)
         return photographers
 
+    def load_wikidata_bbr_fmis_commonscat(self):
+        """
+        Load all bbr/fmis entries in Wikidata and add any commonscats.
+
+        Overrides any mappings found in heritage.
+        """
+        query_props = {'P373': 'commonscat'}
+        mapped_data = self.mappings['commonscat']
+        data = KMBInfo.query_to_lookup(
+            KMBInfo.build_query('P1260', optional_props=query_props.keys()),
+            props=query_props)
+
+        for k, v in data.iteritems():
+            if v.get('commonscat'):
+                entry = {'wd': v.get('wd'), 'cat': v.get('commonscat')}
+                prefix, _, idno = k.rpartition('/')
```

Comment:
The P1260 Property allows you, per its specification, to input values with an extra `html` parameter, like `raa/fmi/html/10028200010001`, which would not be parsed properly here. On the other hand, [it's never used](https://query.wikidata.org/#SELECT%20%3Fitem%20%3FitemLabel%20%3Fvalue%0A%7B%0A%09%3Fitem%20wdt%3AP1260%20%3Fvalue%20FILTER%20CONTAINS%28str%28%3Fvalue%29%2C%20%22html%22%29%20.%0A%09SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22sv%22%20%20%7D%20%20%20%20%0A%7D%0A), so in practice it's a non-problem.